optimum/habana/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -96,6 +101,60 @@ def retrieve_timesteps(
     return timesteps, num_inference_steps
 
 
+# Copied from diffusers.models.unet_2d_condition.UNet2DConditionModel.set_attn_processor
```

Comment:
Let's remove that and rather put it in the docstring below. Can you also add to the docstring how this method differs from the original one please?

Reply:
done